freqtrade/analyze.py

```diff
@@ -2,270 +2,32 @@
 Functions to analyze ticker data with indicators and produce buy and sell signals
 """
 import logging
-from datetime import datetime
-from enum import Enum
-from typing import Dict, List, Tuple
-
-import arrow
 from pandas import DataFrame, to_datetime
 
-from freqtrade import constants
-from freqtrade.exchange import Exchange
-from freqtrade.persistence import Trade
-from freqtrade.strategy.resolver import IStrategy, StrategyResolver
-
 logger = logging.getLogger(__name__)
 
 
-class SignalType(Enum):
-    """
-    Enum to distinguish between buy and sell signals
-    """
-    BUY = "buy"
-    SELL = "sell"
-
-
-class Analyze(object):
+def parse_ticker_dataframe(ticker: list) -> DataFrame:
```

Comment:
what's the point in leaving a file for only one function? maybe we should find a "better" place for this ... or rename the file to "helper_functions_without_dependencies.py" ... (yes, i know I'm not good at naming things)

Reply:
Renamed `analyze.py` to `exchange_helpers.py` becauce it only parses exchange ticker data.

Reply:
eventually we could move it to the exchange file too ... but that's a topic for another day :)